oracle-plugin/src/test/java/io/cdap/plugin/oracle/OracleSourceDBRecordUnitTest.java

```diff
@@ -95,6 +95,55 @@ public void validatePrecisionLessDecimalParsing() throws Exception {
     Assert.assertEquals(record.get("ID4"), "123.4568");
   }
 
+  /**
+   * Validate the precision less Numbers handling against following use cases.
+   * 1. Ensure that for Number(0,-127) non nullable type a String is returned by default.
+   * 2. Ensure that for Number(0,-127) non nullable type a String is returned,
```

Comment:
this javadoc seems incorrect, should be returning a BigDecimal in the cases where the schema is defined as a Decimal.

Reply:
Updated the comments